src/search/search-box/search-box.stories.tsx

```diff
@@ -45,38 +46,38 @@ const searchOptions: Omit<ProductSearchOptions, 'q'> = {
   rows: 2,
   start: 0,
   url: 'https://example.com',
-  'facet.version': '3.0'
+  'facet.version': '3.0',
 };
 
 export const Basic: Story = {
   render: (args) => {
     return (
-      <SearchContextProvider>
+      <LimitlessUIProvider>
```

Comment:
We may have to revisit this usecase, if the searchbox will ever be used this way. The Autosuggest provider could be relevant to wrap the searchbox, but the search page is just one of many pages on a site and it's more likely the consumer will use the hook to do the search and then pass the search response to a results renderer. Let's revisit in a future PR as we add theming to this and start to use this in the code samples

Reply:
I'm honestly not sure what it is that you are attempting to indicate. This basic story is just about using the searchbox without autosuggest, which seems to me quite relevant as its a separate feature and a separate API that might or might not be set up for a specific customer right?

Reply:
Lets sync over Slack